paper/paper.md

```diff
@@ -0,0 +1,183 @@
+   tools.
+
+To make use of these benefits, datasets must first be converted to the
+BIDS format. Furthermore, already existing data analysis software must be
+extended to be aware of the BIDS format and how to properly use it.
+
+This is where ``MNE-BIDS`` steps in. Harnessing BIDS and the data analysis
+software MNE-Python [@Agramfort2013], it is the goal of ``MNE-BIDS`` to
+automatically organize raw MEG, EEG, and iEEG datasets into the BIDS format and
+to facilitate their analysis.
+
+For example, starting with a single directory full of data files with arbitrary
+names, ``MNE-BIDS`` can be used to extract existing metadata, reorganize the
+files into the BIDS format, and write additional metadata. Moreover,
+``MNE-BIDS`` supports converting raw data formats that are not BIDS
+compatible into permissible formats. As a result, users can easily convert
+their datasets to BIDS in a matter of minutes, rather than hours of manual
+labour. All conversions performed by MNE-BIDS are validated with the
+[bids-validator](https://github.com/bids-standard/bids-validator) to ensure
+compatibility with the BIDS format.
+
+In addition to this core functionality, ``MNE-BIDS`` is continuously being
+extended to facilitate the analysis of BIDS formatted data.
+To name some features, it is possible to read a raw BIDS dataset and obtain a
+Python object, ready for analyis with MNE-Python.
+Users can save a T1-weighted anatomical MRI image alongside the MEG or EEG data
+and apply an automatic defacing algorithm for anonymization purposes.
+As a last example, fiducial points can be saved to store coregistration
+information between the electrophysiology and MRI data, which allows for
+automatic computation of forward and inverse solutions.
+
+Users can easily install ``MNE-BIDS`` on all majorplatforms via `pip` and
+`conda`, and its functionality is continuously tested on Windows, OSX, and
+Linux.
+Other than three core dependencies for scientific computing
+(`numpy`, `scipy`) and handling of neurophysiological data (`mne`),
+``MNE-BIDS`` has minimal dependencies, all of which are optional. The API of
+the package is stable and extensively documented and explained in examples
+(https://mne.tools/mne-bids/).
+
+As of writing, ``MNE-BIDS`` has received code contributions from 15
+contributors and its user base is steadily growing. Code development is
+active (https://github.com/mne-tools/mne-bids/graphs/commit-activity) and the
+developer team is committed to provide timely support for issues opened on the
+GitHub issue tracker.
+
+MNE-BIDS is used as a dependency in several other software packages such as
+the [MNE-study-template](https://github.com/mne-tools/mne-study-template), an
+automated analysis pipeline based on [@Mainak2018], and
+[Biscuit](https://github.com/Macquarie-MEG-Research/Biscuit), a graphical
+user interface to format BIDS data.
+Lastly, several large institutions have adopted MNE-BIDS for their
```

Comment:
@alexrockhill can you confirm this?

Reply:
I was basing this off an email that an employee requested MNE-BIDS to be allowed in the Martinos Python distribution. (Email from 3rd of June)